pkg/local/utils.go

```diff
@@ -0,0 +1,251 @@
+package local
```

Comment:
Naive question, there's no way to reuse existing functions from the docker executor to do stuff like fetch logs, wait pull image, fetch container, etc.? 
At the end of the day it's still a docker container (that's the naive part of me speaking 🍭 )

Reply:
yes that's correct, but somethings can't be used in the network eg labels
but it can be used in --local mode
I will modify thoose functions